data/35e/wizards_of_the_coast/supplement/savage_species/ss_abilities.lst

```diff
@@ -122,3 +122,9 @@ Vampire Energy Drain	DESC:Your slam attack (or any other natural weapon you poss
 
 Create Spawn	DESC:A humanoid or monstrous humanoid slain by your energy drain attack rises as a vampire spawn after 1d4 days after burial. If you've drained its Con score to 0 or lower, it returns as a spawn if it had 4 or fewer HD, and as a vampire if it had 5 or more HD. In either case, its under your command.You may have no more HD of enslaved spawn than your own HitDice.|PREVAREQ:VampireCreateSpawnLVL,1	DESC:A humanoid or monstrous humanoid slain by your energy drain attack rises as a vampire spawn after 1d4 days after burial. If you've drained its Con score to 0 or lower, it returns as a spawn if it had 4 or fewer HD, and as a vampire if it had 5 or more HD. In either case, its under your command.You may have no more HD of enslaved spawn than twice your own HitDice.|PREVAREQ:VampireCreateSpawnLVL,2	CATEGORY:Special Ability
 #VampireCreateSpawnLVL 1-2
+
+# Grig - defined in MM already, but partiall overridden here
+Spell-Like Abilities Grigg			KEY:Grig ~ Spell-Like Abilities				CATEGORY:Special Ability	TYPE:SpecialAttack																		VISIBLE:YES																					DESC:%1/day - disguise self, entangle (DC 13), invisibility (self only), pyrotechnics (DC 14), ventriloquism (DC 13). Caster level 9th. The save DCs are Charisma-based.|GriggSpellsPerDay
+#Fiddle					KEY:Grig ~ Fiddle							CATEGORY:Special Ability	TYPE:SpecialAttack.Supernatural																VISIBLE:YES											SPELLS:Magic Item|TIMES=1|CASTERLEVEL=8|Irresistible Dance	DESC:One grig in each band carries a tiny, grig-sized fiddle. When the fiddler plays, any nonsprite within 30 feet of the instrument must succeed on a DC 12 Will save or be affected as though by Otto's Irresistible Dance for as long as the playing continues. The save DC is Charisma-based.
```

Comment:
Why are we introducing commented out code?

Reply:
removed, I had temporarily disabled bits while testing, and now have fully removed them